q2_moshpit/kraken2/bracken.py

```diff
@@ -111,12 +111,34 @@ def _assert_read_lens_available(
         )
 
 
+def _add_unclassified(
```

Comment:
This needs at least one comment explaining what the function does. Maybe add some minor comments inside the function, though they are not necessary.